src/tests/Microsoft.VisualStudio.Composition.Tests/XunitExtensions/MefFactDiscoverer.cs

```diff
@@ -70,6 +70,11 @@ public MefFactTestCase(IMessageSink diagnosticMessageSink, TestMethodDisplay def
                 this.compositionVersions = factAttribute.CompositionVersions;
                 this.noCompatGoal = factAttribute.NoCompatGoal;
                 this.invalidConfiguration = factAttribute.InvalidConfiguration;
+
+                if (factAttribute.SkipOnMono && Type.GetType("Mono.Runtime") != null)
```

Comment:
I haven't found such a way. As far as I can tell it can only see its own attribute, not the other attributes. If you find a way, let me know.